Mathlib/Algebra/GroupPower/Basic.lean

```diff
@@ -83,12 +83,12 @@ theorem pow_one (a : M) : a ^ 1 = a := by rw [pow_succ, pow_zero, mul_one]
 #align pow_one pow_one
 
 /-- Note that most of the lemmas about powers of two refer to it as `sq`. -/
-@[to_additive two_nsmul ""]
-theorem pow_two (a : M) : a ^ 2 = a * a := by rw [pow_succ, pow_one]
+@[to_additive two_nsmul] lemma pow_two (a : M) : a ^ 2 = a * a := by rw [pow_succ, pow_one]
```

Comment:
I think the empty docstring on `two_nsmul` is intentional and should not be removed, since the "sq" comment is irrelevant there.

Reply:
That's bad practice. If you want to shut the `to_additive_doc` linter (which hasn't even been ported), you should tag with `nolint to_additive_doc`, not placate an empty docstring.

Reply:
I misunderstood or misremembered how this worked; please disregard.